developers/weaviate/concepts/indexing/inverted-indexes.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Inverted indexes
+sidebar_position: 100
+image: og/docs/indexing.jpg
+# tags: ['basics']
+---
+
+[Inverted indexes](https://en.wikipedia.org/wiki/Inverted_index) map the contents of documents to improve search performance. The Weaviate database uses inverted indexes to search object properties efficiently.
+
+There are three inverted index types in Weaviate:
+
+- [`indexSearchable`](#indexsearchable) A default index. BM25 (keyword) and hybrid search use `indexSearchable`.
```

Comment:
I would say "enabled by default"; "A default index" makes it sound like an one-of choice, like hnsw/flat/dynamic